src/Microsoft.VisualStudio.ProjectSystem.VisualBasic/ProjectSystem/LanguageServices/VisualBasicCommandLineParserProvider.cs

```diff
@@ -13,11 +14,21 @@ public VisualBasicCommandLineParserProvider()
         {
         }
 
-        [Export(typeof(CommandLineParser))]
+        [Export(typeof(IParseCommandLineArguments))]
         [AppliesTo(ProjectCapability.VisualBasic)]
-        public CommandLineParser Parser
+        public IParseCommandLineArguments Parser
         {
-            get { return VisualBasicCommandLineParser.Default; }
+            get { return new VisualBasicParseCommandLineArguments(); }
+        }
+
+        internal class VisualBasicParseCommandLineArguments : IParseCommandLineArguments
+        {
+            private VisualBasicCommandLineParser _parser = VisualBasicCommandLineParser.Default;
```

Comment:
Why do you even need a field here - why can you just call through Default property in the Parse method?